cypress/integration/userCanSeeLogInLandingPage.feature.js

```diff
@@ -0,0 +1,50 @@
+describe('User can see login landing page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  describe('Successfully', () => {
+    before(() => {
+      cy.intercept(
+        'POST',
+        'https://fake-newzzzz.herokuapp.com/api/auth/sign_in',
+        {
+          fixture: 'handleLogin.json',
+        }
+      );
+    });
+
+    it('is expected to show dashboard after login', () => {
+      cy.get('[data-cy=login-form]').should('exist');
+      cy.get('[data-cy=login-form]').within(() => {
+        cy.get('[data-cy=login-username]').type('user@mail.com');
+        cy.get('[data-cy=login-password]').type('password');
+        cy.get('[data-cy=login-btn]').click();
+      });
+      cy.get('[data-cy=create-article-btn]').should('be.visible');
+    });
+  });
+
```

Comment:
Could we display the name of the user?

Reply:
It is not really in scope now, so we will do it in the next PR